CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ IMPROVEMENTS:
     - tweak params
     - only process one block at a time to avoid starving
 - [common] bit array functions which take in another parameter are now thread safe
+- [common] SplitAndTrim was deleted
```

Comment:
Its kind of confusing, but we put changelog updates for the next release in `CHANGELOG_PENDING.md`, could you move this there, and put it in the breaking changes section?